Functions/InModuleScope.ps1

```diff
@@ -108,7 +108,7 @@ function Get-ScriptModule
         throw "No module named '$ModuleName' is currently loaded."
     }
 
-    $scriptModules = @($modules | & $SafeCommands['Where-Object'] { $_.ModuleType -eq 'Script' })
+    $scriptModules = @($modules | & $SafeCommands['Where-Object'] { $_.ModuleType -in @( 'Script', 'Manifest' ) })
```

Comment:
Does the 'ScriptModules' have to be renamed due to changed usage?

Reply:
But generally, your pull request looks reasonable. Please rebase it after merging #925 to master.

Thank you for your contribution!